### DISCUSSION TITLE
Can i use your package in 2024

### DISCUSSION THREAD

**Discussion by Trung15010802** [fzyzcjy/flutter_smooth#196] [Q&A] [answered]
And how i can use your custom flutter framework? Can you make a video?

**Comment by fzyzcjy** [comment, marked as answer]
Hi, the state is still same as before, thus not ready for apps yet. (I am busy with https://github.com/fzyzcjy/flutter_rust_bridge etc recently)

**Reply by Trung15010802** [reply]
Don't forget this project 🥰

**Comment by escamoteur** [comment]
what is the current state of the PRs into the main Flutter framework. I just stumbled upon this and wonder why this isn't added to the framework yet

**Reply by fzyzcjy** [reply]
Hi, it is still the same state yet. However, hixie had some new discussions a few months ago in https://github.com/flutter/flutter/issues/101227

**Comment by Trung15010802** [comment]
Hi @fzyzcjy , what do you think about this? 
https://flutterfoundation.dev/blog/posts/we-are-forking-flutter-this-is-why/
